src/main/java/io/confluent/kafkarest/SimpleConsumerConfig.java

```diff
@@ -0,0 +1,41 @@
+package io.confluent.kafkarest;
+
+import kafka.consumer.ConsumerConfig;
+
+import java.util.Properties;
+
+public class SimpleConsumerConfig {
+
+  final ConsumerConfig consumerConfig;
+
+  public SimpleConsumerConfig(final Properties originalProperties) {
```

Comment:
Another way to get at these constants is to just use the values from the ConsumerConfig object directly. In Java that'll look a bit odd, something like `kafka.consumer.ConsumerConfig$.MODULE$.MaxFetchWaitMs()` for the fetchWaitMaxMs setting, but it gets rid of the need to instantiate the ConsumerConfig at all.

A lot of this will also get simpler with the new consumer, which has a better config class with default values defined as static constants that are easily accessible.